packages/connectivity/src/scp-cf/destination/destination-service-types.ts

```diff
@@ -238,14 +238,44 @@ export type DestinationRetrievalOptions = CachingOptions & {
   retry?: boolean;
 };
 
+/**
+ * Destination for HTTP request where the URL is mandatory.
+ */
+export type HttpDestination = Destination & { url: string };
+
+/**
+ * Assertion that the Destination is a HttpDestination.
+ * This method comes in handy when you retrieved a destination from the destination service and need to check if it is a HttpDestination.
+ * @param destination - Destination or HttpDestination.
+ */
+export function assertHttpDestination(
+  destination: Destination | HttpDestination
+): asserts destination is HttpDestination {
+  if (!isHttpDestination(destination)) {
+    if (!destination.url) {
+      throw new Error(
+        `The 'url' property is not set for destination ${destination.name} which is mandatory if you use it as an 'HTTP destination`
+      );
+    }
+    throw new Error(
+      `The 'type' property is  ${destination.type} instead of  HTTP for destination '${destination.name}' which is mandatory if you use it as an 'HTTP destination`
+    );
```

Comment:
```suggestion
    throw new Error(destination.url ? 
      `The 'type' property is  ${destination.type} instead of  HTTP for destination '${destination.name}' which is mandatory if you use it as an 'HTTP destination` : `The 'url' property is not set for destination ${destination.name} which is mandatory if you use it as an 'HTTP destination`
    );
```
[pp]